app/views/questions/work_situations/edit.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :page_title, t('views.questions.work_situations.title', :year => current_intake.most_recent_filing_year, :count => current_intake.filing_joint_yes? ? 2 : 1) %>
+
+<% content_for :form_card do %>
+  <%= form_with model: @form, url: current_path, local: true, method: "put", builder: VitaMinFormBuilder, html: { class: "form-card" } do |f| %>
+    <h1 class="form-question spacing-below-35">
+      <%= content_for(:page_title) %>
+    </h1>
+    <p>
+      <%=t('views.questions.work_situations.help_text') %>
+    </p>
+    <div class="form-card__stacked-checkboxes spacing-above-0">
+      <%= f.cfa_checkbox(:had_wages, t('views.questions.work_situations.options.had_wages_html', :count => current_intake.filing_joint_yes? ? 2 : 1), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:had_self_employment_income, t('views.questions.work_situations.options.had_self_employment_income_html', :count => current_intake.filing_joint_yes? ? 2 : 1), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:had_unemployment_income, t('views.questions.work_situations.options.had_unemployment_income_html', :count => current_intake.filing_joint_yes? ? 2 : 1), options: { checked_value: "yes", unchecked_value: "no" }) %>
+      <%= f.cfa_checkbox(:had_tips, t('views.questions.work_situations.options.had_tips', :count => current_intake.filing_joint_yes? ? 2 : 1), options: { checked_value: "yes", unchecked_value: "no" }) %>
```

Comment:
Would it be cleaner to add a new method to the Intake class?

```ruby
def filer_count
  current_intake.filing_joint_yes? ? 2 : 1
end
```

```ruby
t('views.questions.work_situations.options.had_wages_html', :count => current_intake.filer_count)
```

Reply:
yeah, i can refactor that. used widely though not just here, so will expand the ticket. maybe as a follow-up?

Reply:
That sounds good. I didn't realize it was a pattern used elsewhere.